_config.yml

```diff
@@ -58,31 +58,31 @@ t:
         es: "Por ejemplo"
     what-do-sea-otters-eat:
         en: "What do sea otters (<em>Enhydra lutris</em>) eat?"
-        es: "Que come la nutria (<em>Enhydra lutris</em>)?"
+        es: "Qué come la nutria (<em>Enhydra lutris</em>)?"
     or:
         en: "or"
         es: "o"
     what-do-honey-bees-pollinate:
-        en: "What do honey bees (Apis) pollinate?"
-        es: "Que plantas poliniza la abeja mielera (Apis)?"
+        en: "What do honey bees (<em>Apis</em>) pollinate?"
```

Comment:
@AntarMijail I think that genus names are not italic, but binomial names are. From https://en.wikipedia.org/wiki/Binomial_nomenclature#Other_ranks - ```Ranks above genus (e.g., family, order, class) receive one-part names, which are conventionally not written in italics. ``` . Please let me know if you object, otherwise, I'd be happy to remove the italics for the genus name.

Reply:
Hi @jhpoelen like you accurate mark, all ranks above genus never been italic but always the Genera and the specific epithet, also if the Genera are alone we need to italicize the word because it is part of the Scientific name (e.g. in the wiki link you can find ...among botanists as to whether the genera <em>Chionodoxa</em> and <em>Scilla</em> are sufficiently different for them to be kept separate... or  ..."<em>Canis</em> sp." means "an unspecified species of the genus <em>Canis</em>", while "<em>Canis</em> spp." means "two or more species of the genus <em>Canis</em>")  On this two examples the authors refer a particular genus of one or more species, and the genera always are italic. Furthermore, in our particular case, the accurate words need to be <em>Apis</em> spp. because are two or more unspecified species of the genus <em>Apis</em> doing the action, also if we write on this way we maintain the binomial convention. 

I hope to be clear, but if you have more doubts about it or I am not convincing we can continue talking about it

Reply:
Thanks for clarifying!